samples/apps/copilot-chat-app/webapi/CopilotChat/Skills/ChatSkills/DocumentMemorySkill.cs

```diff
@@ -41,13 +41,16 @@ public class DocumentMemorySkill
     /// Query the document memory collection for documents that match the query.
     /// </summary>
     /// <param name="query">Query to match.</param>
-    /// <param name="context">The SkContext.</param>
+    /// <param name="chatId">ID of the chat that owns the documents.</param>
+    /// <param name="tokenLimit">Maximum number of tokens.</param>
+    /// <param name="textMemory">Semantic text memory.</param>
+    /// <returns>A string containing snippets from the relevant documents.</returns>
     [SKFunction, Description("Query documents in the memory given a user message")]
     public async Task<string> QueryDocumentsAsync(
```

Comment:
Need to update error handling when this is called. Currently checking context.ErrorOccurred, which is not populated when called directly as a normal C# function (as opposed to being called by the kernel)